modules/gitops-enabling-the-creation-of-aggregated-cluster-roles.adoc

```diff
@@ -0,0 +1,270 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-enabling-the-creation-of-aggregated-cluster-roles_{context}"]
+= Enabling the creation of aggregated cluster roles
+
+To enable the creation of aggregated cluster roles for the Argo CD Application Controller component of a cluster-scoped Argo CD instance, you must configure the corresponding field by editing the YAML file of the Argo CD custom resource (CR).
+
+.Procedure
+
+. In the Argo CD CR, set the value of the `.spec.aggregatedClusterRoles` field to `true`:
++
+.Example Argo CD CR
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example # <1>
+  namespace: spring-petclinic # <2>
+# ...
+spec:
+  aggregatedClusterRoles: true # <3>
+# ...
+----
+<1> The name of the cluster-scoped instance.
+<2> The namespace where you want to run the cluster-scoped instance.
+<3> The value set to `true` enables the creation of aggregated cluster roles. If you do not want to enable the creation of aggregated cluster roles, do not include this line or set the value to `false`.
```

Comment:
```suggestion
<3> The value set to `true` enables the creation of aggregated cluster roles. If you do not want to enable the creation of aggregated cluster roles, either do not include this line or set the value to `false`.
```
either/or
neither/nor
OK, I could only find a British citation, but it's the same in the US, https://dictionary.cambridge.org/us/grammar/british-grammar/either-or

Reply:
Addressed